### DISCUSSION TITLE
Drupal 8 image styles problem

### DISCUSSION THREAD

**Discussion by Blinks44** [laravel/valet#1125] [Q&A]
Friends, i have very strange problem with drupal8:
When i upload images through admin panel, drupal creating original image and thumbnail size of this image, but not other image styles (medium, large and custom)... "Google" says, that this is because nginx configuration, is it true and what i could do with it?

![2019-10-16_18-46-07](https://user-images.githubusercontent.com/8570193/66936134-dea0ca80-f045-11e9-9e87-bc5d623c3367.png)

Thank you

**Comment by jrbarnard** [comment]
On a fresh install of Drupal 8 and the latest Valet version, I uploaded an image to the article content type, it created thumbnail and large, but not medium or a custom one I created. On looking at the Drupal8 [docs](https://www.drupal.org/docs/user_guide/en/structure-image-styles.html) I believe this is intended, it will only generate the rest on their first request.
I changed the display image style to my custom image style and then loaded the saved content and it generated as expected.
Can you test on a fresh install of d8 & check if you're on the latest valet version to see if this is an issue?
